tests/acceptance/stepDefinitions/loginContext.js

```diff
@@ -1,13 +1,18 @@
 const { client } = require('nightwatch-api')
-const { Given, Then, When } = require('cucumber')
-// respect custom credentials via env vars
-const OC_USER = process.env.OC_USER || null
```

Comment:
Why did you remove this ? I'm unable to use testing against my instance because my password isn't admin...
Don't want to use docker for every single acceptance test run :unamused:

Reply:
will bring it back then .... maybe moving this to the nightwatch config is much more reasonable .... just like the lauchurl